src/main/kotlin/creator/buildsystem/gradle/GradleBuildSystem.kt

```diff
@@ -86,7 +86,7 @@ class GradleBuildSystem(
     }
 
     companion object {
-        val DEFAULT_WRAPPER_VERSION = SemanticVersion.release(5, 6, 1)
+        val DEFAULT_WRAPPER_VERSION = SemanticVersion.release(7, 1)
```

Comment:
Should bump this to 7.1.1 now.

We should make this into a data file like the platform versions though, so it doesn't require a code change. Out of scope for thiw PR unless you want to do it.

Reply:
We might have to rethink Gradle versions handling in the wizard as a whole I think. The old SpongeGradle plugin for SpongeAPI 7 doesn't work on newer Gradle versions for example.

Same for Java versions, we could set a min/max Java version in each project config, and warn the user about conflicts (max bound in one config lower than lower bound of another config.) Clearly out of the scope if this PR though ^^